1-0-admin-tasks.md

```diff
@@ -182,13 +182,89 @@ Now that your project has been created, continue to make your first samples.
+
+The _Edit Group_ page will display a number of fields that you can fill in.
+1. Enter a unique _Name_. The name must contain only letters or numbers. It
+   cannot contain spaces or punctuation.
+1. Enter a _Description_ of your group.
+1. Select your user as the only member of your group.
+1. Click the _Save_ button at the upper right.
+
+## 3.2 Update your project's permissions 
+
+Return to your new project's page and update the permissions:
+1. Click the _Permissions_ section divider to expand it.
+1. Click the _Allow all internal users access?_ box to deselect it.
+1. Select your new group in both the _Groups (read)_ and _Groups (write)_
+   sections.
+1. Click the _Save_ button at the upper right.
+1. Click the _My Projects_ tab at the top of the page.
+1. Partner with someone else who has completed these steps and compare the
+   projects on your pages. You should be unable to view the other person's
+project.
+
+Return to your new project's page and update the permissions once more:
+1. Click the _Permissions_ section divider to expand it.
+1. Click the _Allow all itnernal users access?_ box to select it.
+1. Click the _Save_ button at the upper right.
+
 <a name="inst"/>
 
 # 4. Adding new instruments
 
+New sequencing machines must be added to MISO and to the notification server 
+so that MISO can be made aware of their output.
+
+## 4.1 Add a new model of sequencer
+
+If MISO does not yet have this type of sequencer in the database, you will need 
+to contact your MISO administrator to put the new model into MISO before you can 
+add a new sequencer of that model.
+
+1. Create a JIRA ticket under the _GSI Common_ project to add the new sequencer 
+model to MISO. Include the following information:
+* Sequencer manufacturer (Platform)
+* Model
+* Standard read lengths you anticipate using for sequencing (_e.g._ 1.250, 2x126)
+
+## 4.1 Add a new sequencer 
+
+1. Click the _My Account_ tab at the top of the page.
+1. Click _Configure_ in the _Sequencing Machines_ section.
+1. Click the _Add Sequencer_ link in the top right corner.
+1. Fill in the fields of the new row that is inserted at the top of the table:
+    * _Name_: enter the serial number (case-sensitive)
+    * _Platform_: select a platform from the dropdown menu
+    * _Hostname_: enter `localhost`
+1. Click _Add_ to save the new sequencer.
+
+## 4.2 Make the notification server aware of the sequencer's output
+
+1. Create a JIRA ticket under the _GSI Common_ project. Give the name of the new 
+sequencing machine and indicate that the purpose of the ticket is to add the 
+sequencing machine's output folder to the notification server properties file.
+
+
 <a name="reports"/>
 
 # 5. Creating project and monthly reports
 
+MISO has some basic reporting capabilities. However, it also seems to contain
+dragons which periodically kill all of MISO when they are disturbed. Please
+avoid the _Reports_ tab so as not to anger the dragons.
```

Comment:
Are you asking me to delete this? I'd prefer to keep it in, as we will get questions about reporting.